DynamicTerrain/documentation/dynamicTerrainDocumentation.md

```diff
@@ -48,7 +48,7 @@ The distance between the points is constant on the width and is different from t
     for (var l = 0; l < mapSubZ; l++) {                 // loop on height points
         for (var w = 0; w < mapSubX; w++) {             // loop on width points
             var x = (w - mapSubX * 0.5) * 5.0;          // distance inter-points = 5 on the width
-            var z = (l - mapSubZ * 0.5) * 2.0;          // distance inter-points = 2 on the width
+            var z = (l - mapSubZ * 0.5) * 2.0;          // distance inter-points = 2 on the height
```

Comment:
`depth` would be better, height of the terrain is in the y coordinate.

Reply:
so, the horizontal plane is in x-z axis, and the actual "height" or the altitude is in the y axis, right? I got a little confused there ig, I still don't understand how the z axis is for depth?

Reply:
Yes, the Y axis is up, x-z is in the horizontal plane. I used "depth" for lack of a better term, but "height" is not the right one.

Reply:
can we write z as "length" instead? it would atleast make it clear that we are talking about the horizontal plane and it's a different axis than x..

Reply:
sure! I fixed it to "length"